src/main/external-resources/renderers/Sony-PlayStation3.conf

```diff
@@ -39,14 +39,15 @@ Supported = f:mp4        v:mp4|h264               a:aac                m:video/m
 Supported = f:wmv        v:wmv|vc1                a:wma                m:video/x-ms-wmv           n:2
 
 # Supported audio formats:
+Supported = f:aac     a:(?!alac).+   n:2             m:audio/x-m4a
+Supported = f:atrac                  n:2             m:audio/x-oma
```

Comment:
@Sami32 Sorry, I don't think like that - the important thing to me is that things gets done (and done right), I don't care who does it. If it's important to you, we can merge your PR first and I'll adapt this one to it.